core/src/main/resources/static/js/envs.js

```diff
@@ -662,44 +662,65 @@ app.controller("envsCtrl", function($scope, $http, $location, $window) {
                         else if($scope.addNewCluster.type === 'kafkaconnect')
                             clusterType = 'KAFKA_CONNECT';
 
-                        var serviceInput = {};
+                        if($scope.addNewCluster.protocol === "PLAINTEXT"){
+                            swal({
```

Comment:
Do we need the whitespace before the punctuation marks? i.e. `abc ?`, `xyz !`

Reply:
Updated.